docs/installation.rst

```diff
@@ -858,13 +858,19 @@ To allow scheduled queries, add the following to your `config.py`:
                     },
                     'start_date': {
                         'type': 'string',
-                        'format': 'date-time',
                         'title': 'Start date',
+                        # date-time is parsed using the Sugar library, see
```

Comment:
I thought we already had some logic or a library that handles relative dates. I am a little iffy about using this library because it looks like it's maintained by one developer and the last version was 5 months ago so I am not sure how often this library is maintained.

Reply:
@khtruong we use `moment.js`, but it can't parse relative dates like these, which is what we really want.